package.json

```diff
@@ -115,6 +117,8 @@
     "build:prod-source-maps": "npm run write-sha && env-cmd ./.env/.env.prod env-cmd ./.env/.env.source-maps env-cmd ./.env/.env.git npm run build && rm -rf build-production && mv build build-production",
     "build:mobile-stage": "env-cmd ./.env/.env.mobile npm run build:stage && rm -rf build-mobile-staging && mv build-staging build-mobile-staging",
     "build:mobile-prod": "env-cmd ./.env/.env.mobile npm run build:prod && rm -rf build-mobile-production && mv build-production build-mobile-production",
+    "build:ipfs-stage": "env-cmd ./.env/.env.ipfs.stage npm run build:stage && rm -rf build-mobile-staging && mv build-staging build-mobile-staging",
```

Comment:
I don't think you want the `-mobile-` here?

Reply:
greattt catch ty